### PR TITLE
Add requirements to Accumulo connector

### PR DESCRIPTION
**Pull request by Jessie212**

### CODE REVIEW

File: docs/src/main/sphinx/connector/accumulo.rst [hunk 1]
```diff
@@ -24,6 +24,15 @@ JAR file to Accumulo's ``lib/ext`` directory on each TabletServer node.
...
     # TabletServer should pick up new JAR files in ext directory, but may require restart
 
+Requirements
+------------
+
+To connect to Accumulo, you need:
+
+* Accumulo version 2.0.1 or higher.
+* Network access from the Trino coordinator and workers to the Accumulo server.
+  Port ??? is the default port.
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/accumulo.rst:34]: 2181

File: docs/src/main/sphinx/connector/accumulo.rst [hunk 2]
```diff
@@ -24,6 +24,15 @@ JAR file to Accumulo's ``lib/ext`` directory on each TabletServer node.
 
     # TabletServer should pick up new JAR files in ext directory, but may require restart
 
+Requirements
+------------
+
+To connect to Accumulo, you need:
+
+* Accumulo version 2.0.1 or higher.
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/accumulo.rst:32]: The accumulo dependency version is 1.7.4 .. lets use that for now

File: docs/src/main/sphinx/connector/accumulo.rst [hunk 3]
```diff
@@ -24,6 +24,15 @@ JAR file to Accumulo's ``lib/ext`` directory on each TabletServer node.
...
+* Network access from the Trino coordinator and workers to the Accumulo server.
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/accumulo.rst:33]: to the Accumulo Zookeeper server.

**Review by mosabua — CHANGES_REQUESTED** [review]

**Review by mosabua — APPROVED** [review]
Looks good now.

**Review by mosabua — APPROVED** [review]

File: docs/src/main/sphinx/connector/accumulo.rst (hunk 2 above)
Comment (adamjshook) [reply docs/src/main/sphinx/connector/accumulo.rst]: Yes, Accumulo 2.x actually isn't compatible with this connector. I believe it will need split into a separate module/connector with separate client libraries to support 2.x.

File: docs/src/main/sphinx/connector/accumulo.rst [hunk 4]
```diff
@@ -24,6 +24,15 @@ JAR file to Accumulo's ``lib/ext`` directory on each TabletServer node.
 
     # TabletServer should pick up new JAR files in ext directory, but may require restart
 
+Requirements
+------------
+
+To connect to Accumulo, you need:
+
+* Accumulo version 1.7.4 or higher.
```
Comment (hashhar) [inline docs/src/main/sphinx/connector/accumulo.rst:32]: Doesn't work with Accumulo 2.x IIRC.

Comment (adamjshook) [reply docs/src/main/sphinx/connector/accumulo.rst:32]: Correct, doesn't work with 2.x. Relates to https://github.com/trinodb/trino/issues/7297

Comment (mosabua) [reply docs/src/main/sphinx/connector/accumulo.rst:32]: Maybe reword to 

```
Accumulo versions 1.x starting with 1.7.4. Versions 2.x are not supported.
```

**Review by m57lyra — COMMENTED** [review]
LGTM when version fixed

File: docs/src/main/sphinx/connector/accumulo.rst [hunk 5]
```diff
@@ -24,6 +24,15 @@ JAR file to Accumulo's ``lib/ext`` directory on each TabletServer node.
...
+* Network access from the Trino coordinator and workers to the Accumulo
+  Zookeeper server. Port 2181 is the default port.
```
Comment (jhlodin) [inline docs/src/main/sphinx/connector/accumulo.rst:34]: Assuming this means 2181 is the port for the Zookeeper server, but would like that made explicit. "...is the default port for the Zookeeper server"

**Review by jhlodin — CHANGES_REQUESTED** [review]
Minor wording nit, otherwise LGTM

File: docs/src/main/sphinx/connector/accumulo.rst (hunk 5 above)
Comment (Jessie212) [reply docs/src/main/sphinx/connector/accumulo.rst:34]: Repeating would be repetitive. CC @mosabua

**Review by mosabua — APPROVED** [review]
Good to go now.

**Review by hashhar — APPROVED** [review]
